pr_agent/agent/pr_agent.py

```diff
@@ -13,23 +13,20 @@ def __init__(self):
         pass
 
     async def handle_request(self, pr_url, request) -> bool:
-        if any(cmd in request for cmd in ["/answer"]):
+        action, *args = request.split(" ")
+        if any(cmd == action for cmd in ["/answer"]):
             await PRReviewer(pr_url, is_answer=True).review()
-        elif any(cmd in request for cmd in ["/review", "/review_pr", "/reflect_and_review"]):
+        elif any(cmd == action for cmd in ["/review", "/review_pr", "/reflect_and_review"]):
             if settings.pr_reviewer.ask_and_reflect or "/reflect_and_review" in request:
                 await PRInformationFromUser(pr_url).generate_questions()
             else:
-                await PRReviewer(pr_url).review()
-        elif any(cmd in request for cmd in ["/describe", "/describe_pr"]):
+                await PRReviewer(pr_url, args=args).review()
+        elif any(cmd == action for cmd in ["/describe", "/describe_pr"]):
             await PRDescription(pr_url).describe()
-        elif any(cmd in request for cmd in ["/improve", "/improve_code"]):
+        elif any(cmd == action for cmd in ["/improve", "/improve_code"]):
             await PRCodeSuggestions(pr_url).suggest()
-        elif any(cmd in request for cmd in ["/ask", "/ask_question"]):
-            pattern = r'(/ask|/ask_question)\s*(.*)'
-            matches = re.findall(pattern, request, re.IGNORECASE)
-            if matches:
-                question = matches[0][1]
-                await PRQuestions(pr_url, question).answer()
+        elif any(cmd == action for cmd in ["/ask", "/ask_question"]):
+            await PRQuestions(pr_url, args).answer()
         else:
             return False
```

Comment:
**Suggestion:** Consider using a dictionary to map actions to their corresponding functions. This would make the code more readable and maintainable.
```suggestion
        action_map = {
            '/answer': PRReviewer(pr_url, is_answer=True).review,
            '/review': PRReviewer(pr_url, args=args).review,
            '/review_pr': PRReviewer(pr_url, args=args).review,
            '/reflect_and_review': PRInformationFromUser(pr_url).generate_questions if settings.pr_reviewer.ask_and_reflect or "/reflect_and_review" in request else PRReviewer(pr_url, args=args).review,
            '/describe': PRDescription(pr_url).describe,
            '/describe_pr': PRDescription(pr_url).describe,
            '/improve': PRCodeSuggestions(pr_url).suggest,
            '/improve_code': PRCodeSuggestions(pr_url).suggest,
            '/ask': PRQuestions(pr_url, args).answer,
            '/ask_question': PRQuestions(pr_url, args).answer
        }

        action, *args = request.split(" ")
        if action in action_map:
            await action_map[action]()
        else:
            return False
```